lib/event_tracer/dynamo_db_client.rb

```diff
@@ -0,0 +1,5 @@
+class DynamoDBClient
```

Comment:
I think this should be wrapped inside `EventTracer` module to avoid polluting global scope